recipes/jalview/build.sh

```diff
@@ -16,10 +16,10 @@ echo -e '#!/usr/bin/env bash\n\
 JALVIEWDIR=$JALVIEWDIR;\n\
 java -Djava.awt.headless=true -Djava.ext.dirs=\$JALVIEWDIR -cp \$JALVIEWDIR/jalview.jar jalview.bin.Jalview \${@};\n\
 ' | tr -d '\\' > $JALVIEWDIR/jalview.sh; 
-chmod 0755 $JALVIEWDIR/jalview.sh; 
+chmod +x $JALVIEWDIR/jalview.sh; 
 
 # link wrapper script
 mkdir -p $PREFIX/bin
-ln -s $JALVIEWDIR/jalview.sh $PREFIX/bin/jalview
-chmod 0755 $PREFIX/bin/jalview; 
+ln -s ../share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM/jalview.sh $PREFIX/bin/jalview
```

Comment:
should this be $JALVIEWDIR?

Reply:
Nope, tried to use a relative rather than an absolute path link to the wrapper.. not good? Done by the system anyway..

Reply:
This links from the build environment to the $PREFIX. If you want to use relative path, you need to change to $PREFIX at first.

Reply:
Mhh.. sounds reasonable.. *grin* will change it back to $PREFIX to avoid confusion...

Funny thing: seems to be somehow (partially?) Working, since the test finds the linked shell script.. strange..